File: Katas/C#/8KYU/Sum Mixed Array.cs
using System;

public class Kata
{
    public static int SumMix(object[] x)
    {
        var answer = 0;
        for (var i = 0; i < x.Length; i++)
        {
            answer += Convert.ToInt32(x[i]);
        }
        return answer;
    }
}

//Other Solutions:

//using static System.Convert;
//using System.Linq;

//class Kata
//{
//    public static int SumMix(object[] x) => x.Sum(ToInt32);
//}

File: Katas/JavaScript/8KYU/Sum Mixed Array.js
function sumMix(x) {
    let answer = 0;
    for (i = 0; i < x.length; i++) {
        answer += parseInt(x[i])
    }
    return answer;
}

//Other Solutions:

//function sumMix(x) {
//    return x.map(a => +a).reduce((a, b) => a + b);
//}